org.flatpak.am-okay.yml

```
@@ -0,0 +1,81 @@
+id: org.flatpak.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --share=network
+    - --socket=x11
+    - --filesystem=host
+    - --filesystem=/tmp
+    - --device=all
+    - --filesystem=/media
+
+modules:
+    - name: am-okay-package
+      buildsystem: simple      
+      sources:
+          - type: git 
+            url: https://github.com/baldeuniversel/am-okay.git 
+            commit: ed3e921ec1241c237887aa9041cad8ac9f0ee3b0 
+
+      build-commands: 
+            - install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/bin/am-okay /app/bin/am-okay
+
+            - install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/share/metainfo/io.github.baldeuniversel.am-okay.metainfo.xml \
+              /app/share/metainfo
+            
+            - |
+              install -D $FLATPAK_BUILDER_BUILDDIR/am-okay/usr/share/icons/hicolor/128x128/apps/org.flatpak.am-okay.png \
+              /app/usr/share/icons/hicolor/128x128/apps
```

Wrong install path for the icons.

```suggestion
              /app/share/icons/hicolor/128x128/apps
```

---

Done !